English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md

```diff
@@ -59,183 +59,307 @@ Transfer, vote, issuance of token, or participation in token offering. Sending s
- 11.5 Function Token transfer. Creates a transaction of token transfer.
+11.1 Interface statement rpc TransferAsset (TransferAssetContract) returns (Transaction){};  
+11.2 Node
+Fullnode. 
+11.3 Parameters
+TransferAssetContract: token name, sender address, recipient address, and the amount of tokens.  
+11.4 Returns
+Transaction: returns transaction of token transfer. Request broadcasting after the transaction is signed by wallet.  
+11.5 Function
+Token transfer. Creates a transaction of token transfer.
 
 ## 12. Participation in token offering
-12.1 Interface statement rpc ParticipateAssetIssue (ParticipateAssetIssueContract) returns (Transaction){};   
-12.2 Node Fullnode.   
-12.3 Parameters ParticipateAssetIssueContract: participant address, issuer address, token name, and amount of token (in sun).   
-12.4 Returns Transaction: returns transaction of participation in token offering. Request broadcasting after the transaction is signed by wallet.   
-12.5 Function Participation in token offering.
+12.1 Interface statement
+rpc ParticipateAssetIssue (ParticipateAssetIssueContract) returns (Transaction){}; 
+12.2 Node
+Fullnode. 
+12.3 Parameters
+ParticipateAssetIssueContract: participant address, issuer address, token name, and amount of token (in sun).  
+12.4 Returns
+Transaction: returns transaction of participation in token offering. Request broadcasting after the transaction is signed by wallet.  
+12.5 Function
+Participation in token offering.
 
 ## 13. Query of all nodes
-13.1 Interface statement rpc ListNodes (EmptyMessage) returns (NodeList) {};   
-13.2 Nodes Fullnode and soliditynode. 13.3 Parameters EmptyMessage: null.   
-13.4 Returns NodeList: returns a list of nodes, including their IPs and ports.   
-14.5 Function Listing the IPs and ports of current nodes.
+13.1 Interface statement
+rpc ListNodes (EmptyMessage) returns (NodeList) {};  
+13.2 Nodes
+Fullnode and soliditynode.
+13.3 Parameters
+EmptyMessage: null.  
+13.4 Returns
+NodeList: returns a list of nodes, including their IPs and ports.  
+13.5 Function
+Listing the IPs and ports of current nodes.
 
 ## 14. Query of list of all tokens
-14.1 Interface statement rpc GetAssetIssueList (EmptyMessage) returns (AssetIssueList) {};   
-14.2 Node Fullnode and soliditynode.   
-14.3 Parameters EmptyMessage: null.   
-14.4 Returns AssetIssueList: AssetIssueContract list containing information on all issued tokens.   
-14.5 Function List of all issued tokens. Display of all issued tokens for user’s reference.
+14.1 Interface statement
+rpc GetAssetIssueList (EmptyMessage) returns (AssetIssueList) {};  
+14.2 Node
+Fullnode and soliditynode.  
+14.3 Parameters
+EmptyMessage: null.  
+14.4 Returns
+AssetIssueList: AssetIssueContract list containing information on all issued tokens.  
+14.5 Function
+Query of list of all issued tokens. Display of all issued tokens for user’s reference.
```

Comment:
To make it clear, add `query of` in the beginning

Reply:
Should be:

Query list of all issued tokens.

Reply:
Thanks @kookiekrak How was `14. Query of list of all tokens`? Do it need to be fixed too?